src/axom/inlet/inlet_utils.cpp

```diff
@@ -12,5 +12,17 @@ void setWarningFlag(axom::sidre::Group* root) {
   }
 }
 
+std::string appendPrefix(const std::string& prefix, const std::string& name)
+{
+  return (prefix == "") ? name : prefix + "/" + name;
+}
+
+std::string removePrefix(const std::string& prefix, const std::string& name) {
+  SLIC_ASSERT_MSG(axom::utilities::string::startsWith(name, prefix), 
+                  fmt::format("Provided name {0} does not contain prefix{1}", 
+                  name, prefix));
+  return name.substr(prefix.size()-1);

```

Comment:
One issue with this approach is that it will only fail in debug builds, and happily remove N characters from your string without and indications of trouble in release builds.

A different approach would be to remove the prefix if it is there, and otherwise just return the original string.

Reply:
Also, it would probably be a good idea to check that `name` has length at least `prefix.size()`.